src/main/resources/assets/lambdynlights/lang/it_it.json

```diff
@@ -0,0 +1,19 @@
+{
+  "lambdynlights.menu.light_sources": "Fonti di Luci Dinamiche",
+  "lambdynlights.menu.title": "Impostazioni di LambDynamicLights",
+  "lambdynlights.menu.canvas.1": "Disattivare luci dinamiche di entità è raccomandato con Canvas!",
+  "lambdynlights.menu.canvas.2": "L'illuminazione dinamica con Canvas è pesante sulla performance per il momento!",
+  "lambdynlights.option.block_entities": "Entità blocco",
+  "lambdynlights.option.entities": "Entità",
+  "lambdynlights.option.mode": "Luci Dinamiche",
+  "lambdynlights.option.water_sensitive": "Controllo di sensibilità dell'acqua",
+  "lambdynlights.toast.first_time": "Disattivato di predefinito, controlla impostazioni della mod o di Grafica!",
+  "lambdynlights.tooltip.block_entities": "Attiva l'illuminazione dinamica delle entità blocco.",
+  "lambdynlights.tooltip.creeper_lighting": "Imposta la modalità di illuminazione dinamica dei Creeper.\n- %s disattiva l'illuminazione dinamica dei Creeper.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
+  "lambdynlights.tooltip.entities": "Attiva l'illuminazione dinamica di entità. Nota: I giocatori sono sempre attivi.",
+  "lambdynlights.tooltip.mode.1": "Attiva le luci dinamiche. Se attivo, un giocatore con una torcia in mano emetterà luce per esempio.",
+  "lambdynlights.tooltip.mode.2": "%s e %s avranno un leggero ritardo ma minimizzeranno gli aggiornamenti di luce.",
+  "lambdynlights.tooltip.mode.3": "%s è illuminazione dinamica soffusa.",
+  "lambdynlights.tooltip.tnt_lighting": "Imposta la modalità di illuminazione dinamica della TNT.\n- %s disattiva l'illuminazione dinamica della TNT.\n- %s imposta una luminosità costante.\n- %s imposta una luminosità dinamica.",
+  "lambdynlights.tooltip.water_sensitive": "Attiva il controllo di fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti in mano non emetteranno luce mentre sommersi in acqua."
```

Comment:
Anche qui le proposizioni sono sbagliate. Inoltre hai aggiunto qualche dettaglio che le stringhe originali non hanno.
"Attiva il controllo **delle** fonti di luce sensibili all'acqua. Questo vuol dire che certi oggetti ~~in mano~~ non emetteranno luce mentre sommersi in acqua."